capability/capability_test.go

```diff
@@ -21,6 +21,16 @@ const (
 	maxLastCap = CAP_CHECKPOINT_RESTORE
 )
 
+func requirePCapSet(t *testing.T) {
```

Comment:
nit: can you use `t testing.TB` here instead (in case we'll add a benchmark later, this function can also be used from it).